mongo/index.go

```diff
@@ -113,22 +132,22 @@ func (mg *Graph) GetVertexTermAggregation(ctx context.Context, label string, fie
 		ag = append(ag, bson.M{"$limit": size})
 	}
 
-	session := mg.ar.session.Copy()
-	defer session.Close()
-	vcol := mg.ar.VertexCollection(session, mg.graph)
-	pipe := vcol.Pipe(ag)
-	iter := pipe.Iter()
-	defer iter.Close()
+	vcol := mg.ar.VertexCollection(mg.graph)
+	cursor, err := vcol.Aggregate(context.TODO(), ag)
+	if err != nil {
+		return nil, err
+	}
 	result := map[string]interface{}{}
-	for iter.Next(&result) {
+	for cursor.Next(context.TODO()) {
+		cursor.Decode(&result)
```

Comment:
None of these aggregation methods are needed.  These were dropped from the Index interface some time ago.